hardware/industrial/README.md

```diff
@@ -29,6 +29,10 @@ pi@raspberrypi:~ $ /opt/vc/bin/vcmailbox 0x00010004 8 8 0 0
 
 The above uses the [mailbox property interface](https://github.com/raspberrypi/firmware/wiki/Mailbox-property-interface) `GET_BOARD_SERIAL` with a request size of 8 bytes and response size of 8 bytes (sending two integers for the request 0, 0). The response to this will be two integers (0x00000020 and 0x80000000) followed by the tag code, the request length, the response length (with the 31st bit set to indicate that it is a response) then the 64 bit serial number (where the MS 32bits are always 0).
 
+## Write and read Customer OTP values
+
+**WARNING: The OTP values are One-Time Programmable, the `SET_CUSTOMER_OTP` tag can only be used once on each row**
```

Comment:
To be exact each bit can individually be switched from 0 to 1 but not the other way around.

You can set a row to 1, then 3, then 0x13, but you can't go back.

One could write a detailed document about this but for this document I think a quick explanation is better to inform the reader before they start experimenting.

Reply:
My assumption (which might be incorrect?) would be that 99.9% of users will never use these Customer OTP values, but the 0.1% who do (I _think_ they're intended for people who are [building a product](https://www.raspberrypi.org/blog/supporting-raspberry-pis-industrial-customers/) around a Raspberry Pi) will probably want to know **exactly** how they work? :thinking:  (or at least not be told incorrect information)

I'm happy to defer to @JamesH65 's or @aallan 's decision though (I'm sometimes too pedantic for my own good :laughing: ).

Reply:
Agree they can be in the same document. To be fully knowledgeable about this area you have to read the whole document, but the risks should be clear much earlier.

 Especially since much learning is about trying, and this feature doesn't allow for testing and starting over.

One way could be to place a big warning, not generic "read the full document", but specific "some commands in this document are irreversible".

Reply:
What about just changing it to:
**WARNING: The OTP values are One-Time Programmable, once a bit has been changed from 0 to 1, it can't be changed back**

:grey_question: 

> this feature doesn't allow for testing and starting over.

At least a Raspberry Pi is relatively inexpensive, and not several hundreds (or thousands) of pounds/dollars :wink:

Reply:
Yes, i like it.